src/wp-includes/class-wp-block-type.php

```diff
@@ -405,3 +405,47 @@ public function get_attributes() {
 			array();
 	}
 }
+
+function _wp_normalize_block_styles_from_mixed_format_to_array( $args ) {
```

Comment:
You can put that function directly in the `WP_Block_Type` class and call from the method that integrates `register_block_type_args` filter.